config/topic_rate_limit_whitelist.txt

```diff
@@ -0,0 +1,5 @@
+#########################################################################################
+# Rate limit whitelist for Fail2Topic. No point using this if fail2topic isn't enabled. #
+# One IP address per line.																#
+#########################################################################################
```

Comment:
This will need all the IPs that are exempt, don't forget to include the host in here